.travis.yml

```diff
@@ -60,33 +60,20 @@ jobs:
       python: '3.7'
       cache: pip
       install:
-        # Initialize services required to run tests
-        - docker-compose up --detach mysql redis rabbitmq
-        - pip install -r requirements/docs.txt # Install separately since it has no hashes
-        - pip install -r requirements/dev.txt
-        - pip install -r requirements/common.txt
+        - pip install tox
       script:
-        - pre-commit run --all-files markdownlint
-        - ./runchecks.sh
-        - ./manage.py check
-        # Several security features in settings.py (eg setting HSTS headers) are conditional on
-        # 'https://' being in the site URL. In addition, we override the test environment's debug
-        # value so the tests pass. The real environment variable will be checked during deployment.
-        - SITE_URL=https://treeherder.dev TREEHERDER_DEBUG=False ./manage.py check --deploy --fail-level WARNING
-        # Exercising running tests outside of Docker
-        - pytest tests/ --ignore=tests/selenium --ignore=tests/extract
-        # This is to deal with running the containers with --detached
-        - docker-compose down
+        - tox
+        - tox -e virtualenv
```

Comment:
I am thinking that running two tox environments will cause the installation of packages twice.
Let's see if we can fold the two together.